news-list/src/components/NewsForm/NewsForm.spec.js

```diff
@@ -0,0 +1,51 @@
+import { mount } from '@vue/test-utils'
+import NewsForm from '@/components/NewsForm/NewsForm.vue'
+
+describe('NewsForm.vue', () => {
+  describe(':buttons', () => {
+    describe('click create', () => {
```

Comment:
Verschachteltes describe? Macht man das so? macht man das nicht mit mehreren it? Woher habt ihr das? Sehen wir so zum ersten Mal.

Reply:
Man kann es auf jeden Fall machen (siehe z.B. https://www.bignerdranch.com/blog/why-do-javascript-test-frameworks-use-describe-and-beforeeach/). Wir werden aber nochmal schauen, ob es in unseren Tests Sinn ergibt